ha/core/event_analyzer/parser/parser.py

```diff
@@ -188,9 +188,10 @@ def parse_event(self, msg: str) -> HealthEvent:
             resource_id = cluster_resource_alert[EventAttr.EVENT_PAYLOAD.value][HealthAttr.RESOURCE_ID.value]
             event_type = cluster_resource_alert[EventAttr.EVENT_PAYLOAD.value][HealthAttr.RESOURCE_STATUS.value]
             specific_info = cluster_resource_alert[EventAttr.EVENT_PAYLOAD.value][HealthAttr.SPECIFIC_INFO.value]
-            if specific_info and specific_info["generation_id"] and source == HEALTH_EVENT_SOURCES.MONITOR.value:
-                generation_id = cluster_resource_alert[EventAttr.EVENT_PAYLOAD.value][HealthAttr.SPECIFIC_INFO.value]["generation_id"]
-                specific_info = {"generation_id": generation_id, "pod_restart": 0}
+            if resource_type == CLUSTER_ELEMENTS.NODE.value:
```

Comment:
Why do you think that this additional change is required?

Reply:
Rechecked https://github.com/Seagate/cortx-ha/pull/662 PR ,
This line missed from last PR : https://github.com/Seagate/cortx-ha/pull/662/files#diff-687536d161c575c3418d6c6ee6badc4afe27aab69b093e8b95767128696dd059L190

Reply:
Ohh Ok..!! Got it.